nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -259,8 +259,9 @@ bool CollisionMonitor::configurePolygons(
   try {
     auto node = shared_from_this();
 
+    // Leave it to be not initialized: to intentionally cause an error if it will not set
     nav2_util::declare_parameter_if_not_declared(
-      node, "polygons", rclcpp::ParameterValue(std::vector<std::string>()));
```

Comment:
I think this is the only non-backportable change to Iron since that would break a configuration for a user

... also @AlexeyMerzlyakov is this right? Didn't we have this be set to nothing so the case that it uses dynamic polygon topics? Or is that still OK